server/cmd/mmctl/commands/bot_e2e_test.go

```diff
@@ -432,3 +432,57 @@ func (s *MmctlE2ETestSuite) TestBotCreateCmdF() {
 		s.Require().Empty(printer.GetErrorLines())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestBotUpdateCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.Run("Update bot without permission", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		cmd.Flags().String("username", "newbot", "")
+		cmd.Flags().Lookup("username").Changed = true
+
+		bot, appErr := s.th.App.CreateBot(s.th.Context, &model.Bot{Username: "testbot", OwnerId: s.th.BasicUser.Id})
+		s.Require().Nil(appErr)
+		defer func() {
+			err := s.th.App.PermanentDeleteBot(s.th.Context, bot.UserId)
+			s.Require().Nil(err)
+		}()
+
+		err := botUpdateCmdF(s.th.Client, cmd, []string{"testbot"})
+		s.Require().Error(err)
+		s.Require().Empty(printer.GetErrorLines())
+	})
+
+	s.RunForSystemAdminAndLocal("Update attributes of the bot", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		cmd.Flags().String("username", "newbot", "")
+		cmd.Flags().String("display-name", "new-display-name", "")
+		cmd.Flags().String("description", "new description", "")
```

Comment:
I wonder if the command should error if neither of the three flags is defined. What do you think?